src/Array/slice.js

```diff
@@ -0,0 +1,10 @@
+let _slice = (array, start, end) => {
+  let origArray = array,
```

Comment:
Naming convention: I've seen the SEPs here prefer to use whole words whenever possible instead of abbreviations. The code we write is for humans, and there's a slightly lower cognitive load when we read whole words. Thus, `originalArray`.

Also, save a line by making your line 1 `let _slice = (originalArray, start, end) => {`
and skipping line 2